baarticle.cls

```diff
@@ -51,13 +51,15 @@
     \RequirePackage{hyperref}
     \RequirePackage[style=alttree,nogroupskip,nonumberlist,nopostdot,nolong,nosuper,nolist]{glossaries}
     
-    % hyperref config
-    \hypersetup{
-    colorlinks=true,
-    linkcolor=blue,
-    filecolor=magenta,      
-    urlcolor=cyan,
-    }
+    \ifBA@simple@linkcoloring
+        % hyperref config
+        \hypersetup{
+            colorlinks=true,
+            linkcolor=blue,
+            filecolor=magenta,      
+            urlcolor=cyan,
+        }
+     \fi

```

Comment:
This should very likely be placed within the definition basimple after the call to `\setkeys`. I feel above `\mktitle` is a good place (https://github.com/Nuckal777/ba-latex-template/blob/rewrite/baarticle.cls#L308). I would expect that the check after the imports is either always true or false right now (but I haven't tested).